pkg/email/send_email.go

```diff
@@ -0,0 +1,38 @@
+package email
+
+import (
+	"gopkg.in/gomail.v2"
+)
+
+type Emailer struct {
+	SMTPHost string
+	Port     int
+	Username string
+	Password string
+	Sender   string
+}
+
+type EmailParams struct {
+	SMTPHost string
+	Port     int
+	Username string
+	Password string
+	Sender   string
+}
+
+func NewEmailer(e EmailParams) Emailer {
+	return Emailer(e)
+}
+
+func (e Emailer) SendEmail(receivers []string, subject string, body string, attachmentFilePath ...string) error {
```

Comment:
```suggestion
func (e Emailer) SendEmail(receivers []string, subject string, body string, attachmentFilePath []string) error {
```
I think it'd be better to use array of string for the attachmentFilePath compared the never-endning fiesca of parameters, and it basically limits our signature modification for future use, due to the fact the parameters are not explicit anymore.